docs/docs/integrations/providers/mlflow_tracking.ipynb

```diff
@@ -205,7 +571,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.12"
+   "version": "3.12.2"
```

Comment:
3.12.2? 
Is this auto-generated or are there requirements on py versions for running these notebooks?

Reply:
oh that automatically picks the python version of my local venv haha, shouldn't be visible from users